extensions/publisher-cves/src/components/icons/ArrowTopRight.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <svg
```

Comment:
This is this SVG:
![CleanShot 2025-06-11 at 17 30 49@2x](https://github.com/user-attachments/assets/9e9adc30-04c7-4f50-852f-a060783b25e6)